internal/sweep/sweep.go

```diff
@@ -52,7 +52,7 @@ func SweepServices(region, t string) error {
 			continue
 		}
 
-		if !hasPrefixAny(s.Name, "test-acc", "test-examples", "k8s-") {
+		if !hasPrefixAny(s.Name, "test-tf", "test-acc", "test-examples", "k8s-") {
```

Comment:
nit: perhaps we should move `k8s-` sweeper to the respective repo

Reply:
sounds like a good idea, but we usually have services with k8s prefix in a different project